### PR TITLE
Woo: 3 of 3 - Add updating capabilities to order list management

### PR DESCRIPTION
**Pull request by AmandaRiu**
PR 3 of 3. This final PR adds the logic needed to fetch orders that have been updated since last downloaded.

In this gif, observe how the status of the second record updates from `processing` to `on-hold`

### CODE REVIEW

**Comment by AmandaRiu** [comment]
Updated branch with all the AndroidX changes from the parent branch.

File: fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.java [hunk 1]
```diff
@@ -789,6 +789,7 @@ private void migrateAddOn(@AddOn String addOnName, SQLiteDatabase db, int oldDbV
                      break;
                 case 68:
                     AppLog.d(T.DB, "Migrating addon " + addOnName + " to version " + (oldDbVersion + 1));
+                    db.execSQL("ALTER TABLE WCOrderModel ADD DATE_MODIFIED TEXT");
```
Comment (malinajirka) [inline fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.java]: I assume you intentionally added this into an existing case as this feature hasn't been released yet, right?

File: plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/store/WCOrderStore.kt [hunk 2]
```diff
@@ -433,7 +434,9 @@ class WCOrderStore @Inject constructor(dispatcher: Dispatcher, private val wcOrd
     }
 
     private fun fetchOrdersByIds(payload: FetchOrdersByIdsPayload) {
-        wcOrderRestClient.fetchOrdersByIds(payload.site, payload.remoteIds)
+        payload.remoteIds.chunked(NUM_ORDERS_PER_FETCH).forEach { idsToFetch ->
```
Comment (malinajirka) [inline plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/store/WCOrderStore.kt]: TIL `chunked` ;)

File: fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.java (hunk 1 above)
Comment (AmandaRiu) [reply fluxc/src/main/java/org/wordpress/android/fluxc/persistence/WellSqlConfig.java]: That is correct!

**Comment by AmandaRiu** [comment]
Thanks for the review @malinajirka !! Merging.

**Pull request merged**